qiskit_machine_learning/circuit/library/raw_feature_vector.py

```diff
@@ -65,19 +65,24 @@ class RawFeatureVector(BlueprintCircuit):
     def __init__(self, feature_dimension: Optional[int]) -> None:
         """
         Args:
-            feature_dimension: The feature dimension and number of qubits.
+            feature_dimension: The feature dimension from which the number of
+                                qubits is inferred as ``n_qubits = log2(feature_dim)``
 
         """
         super().__init__()
 
-        self._num_qubits = None
         self._ordered_parameters = ParameterVector("x")
-
+        self.num_qubits = None
         if feature_dimension:
```

Comment:
```suggestion
        if feature_dimension is not None:
```
I think this really should be an explicit check against None. (I realize this is how it was though but since things are being fixed here)